packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -172,7 +174,7 @@ const start = async (zcf, privateArgs) => {
   const getKeywordForBrand = brand => {
     assert(
       keywordForBrand.has(brand),
-      `Issuer not defined for brand ${brand}; first call addIssuer()`,
+      X`Issuer not defined for brand ${brand}; first call addIssuer()`,
```

Comment:
```suggestion
      X`Issuer not defined for brand ${q(brand)}; first call addIssuer()`,
```
This one is less clear, but I would disclose the brand.

Reply:
done